code/index.html

```diff
@@ -2,21 +2,165 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <title>Project 2: Chipotle magazine</title>
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;700&display=swap');
+    </style>  
   </head>
+
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    <header>  <!-- LOGO AND NAV BAR -->  
+    
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+      
+      <img src = "img/chipotle.png" alt= "Chipotle's logo">
+      
+      <nav>
+        <a href="">Articles</a>
+        <a href="">Recipes</a>
+        <a href="">Workshops</a>
+        <a href="">Trips</a>
+      </nav>
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
+     
+      <!-- THE BIG NEWS CONTENT -->
+
+    <section class="flex-parent">
+  
+      <div class="flex-child pic">
+        <img src="img/foodtour.jpg" alt="Tacos image">
+      </div>
+        
+      <div class= "flex-child txt">
+        <h1> Epic streetfood tour in Mexico city!</h1>
+        <p> An unconventional gastronomic experience 
+        that you will never forget! Lorem ipsum dolor, sit amet consectetur adipisicing elit. adipisicing elit.adipisicing elit.adipisicing elit.
+      </p>
+      </div>
+
     </section>
+        <!-- THE OTHER NEWS CONTENT -->
+
+   <section class="grid-container">
+
+      <div class= "grid-item one">
+        <h5>ARTICLE</h5>  
```

Comment:
Since you've only used a h1 as the big headline, then good practise is to use h2 on the next one. Just a little something to keep in mind!